bundle/src/main/java/com/adobe/acs/commons/wcm/tags/DefineObjects.java

```diff
@@ -0,0 +1,93 @@
+package com.adobe.acs.commons.wcm.tags;
+
+import aQute.bnd.annotation.ProviderType;
+import com.day.cq.commons.LanguageUtil;
+import com.day.cq.wcm.api.components.Component;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.jcr.resource.JcrPropertyMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import tldgen.BodyContentType;
+import tldgen.Tag;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.servlet.jsp.tagext.BodyTagSupport;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by derek on 6/9/16.
+ */
+@ProviderType
+@Tag(bodyContentType = BodyContentType.JSP, value = "defineObjects")
+public class DefineObjects extends BodyTagSupport {
+
+    private static final Logger log = LoggerFactory.getLogger(DefineObjects.class);
+
+    @Override
+    public int doEndTag() {
+        log.info("Starting the doEndTag");
+        getComponentPropertyHome();
+        setMergedProperties();
+        return EVAL_PAGE;
+    }
+
+    private Node getComponentPropertyHome() {
```

Comment:
should rename this to setSitewideProperties and remove return value since it is not use.